### ISSUE TITLE
Is 'attr' in one data In celeba-full-tfr?

### ISSUE THREAD

**Issue by wangherr** [openai/glow#112]
~~~
celeba - 4GB. CelebA-HQ 256x256 dataset, as described in Progressive growing of GAN's. 
For 1024x1024 version (120GB), use celeba-full-tfr.tar while downloading.
~~~

In celeba-4GB，there is {‘shape’， ‘data’， ’label‘} in one data, I can't use celeba's attribution to train the model.
![图片](https://user-images.githubusercontent.com/68674487/169331135-94bd6d11-b47f-42c5-8694-b2c824b8822f.png)

In celeba-full-tfr, is 'attr' in one data? 

In other words,  to use celeba's attribution in 256x256x3, is there a prepared resource or need I generate it myself?

Thanks!
